TeaLeaf/src/com/tealeaf/TeaLeaf.java

```diff
@@ -310,36 +311,11 @@ public void onCreate(Bundle savedInstanceState) {
 		int height = display.getHeight();
 		int orientation = getRequestedOrientation();
 
-		// gets real screen dimensions without nav bars on recent API versions
-		if (isFullScreen && Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT && !notched_device) {
-			try {
-				display.getRealSize(screenSize);
-				width = screenSize.x;
-				height = screenSize.y;
-			} catch (NoSuchMethodError e) {}
-		} else if (notched_device) {
-			try {
-				display.getSize(screenSize);
-				width = screenSize.x;
-				height = screenSize.y;
-			} catch (NoSuchMethodError e) {
-			}
-		}
-
-		// flip width and height based on orientation
-		if ((orientation == ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE && height > width)
-			|| (orientation == ActivityInfo.SCREEN_ORIENTATION_PORTRAIT && width > height))
-		{
-			int tempWidth = width;
-			width = height;
-			height = tempWidth;
-		}
-
 		final AbsoluteLayout absLayout = new AbsoluteLayout(this);
```

Comment:
Please add a TODO for landscape. You mentioned, it was not verifed or it might break

Reply:
Done!